manifests/config.pp

```diff
@@ -206,6 +206,7 @@
 
       if $foreman::ipa_manage_sssd {
         $sssd = pick(fact('foreman_sssd'), {})
+        $sssd_default_domain_suffix = $facts['networking']['domain']
```

Comment:
My biggest concern here is where the domain does not match the realm. For example, you may have `foreman.lab.example.com` joined to the `example.com` realm. `$facts['networking']['domain']` would then be `lab.example.com`, but is that really correct? I'm not too familiar with this, so feel free to correct me if I'm wrong.

Reply:
@ekohl, I think the default realm configured can be identified from the `/etc/ipa/default.conf` file. This file exists in both the authentication methods i.e. integration with IPA and AD (specifically we ask users to configure this file in direct AD auth using gssproxy https://docs.theforeman.org/3.0/Administering_Red_Hat_Satellite/index-foreman-el.html#sect-Administering-Configuring_External_Authentication-Using_Active_Directory).

Another way is to rely on end users to provide this value as a installer parameter in order to configure it.

Reply:
> Another way is to rely on end users to provide this value as a installer parameter in order to configure it.

Right now I'm leaning in that direction.